File: src/classes/api/req_res/add-tag-response.ts
'use strict';

import { Tag } from '@/classes/datas/tag';
import { GkillAPIResponse } from '../gkill-api-response';


export class AddTagResponse extends GkillAPIResponse {


    added_tag: Tag;

    constructor() {
        super()
        this.added_tag = new Tag()
    }


}

File: src/server/gkill/api/req_res/add_tag_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type AddTagResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	AddedTag *reps.Tag `json:"added_tag"`
}
